webapp/tests/karma/ts/modules/reports/reports-content.component.spec.ts

```diff
@@ -88,7 +93,9 @@ describe('Reports Content Component', () => {
           FormIconPipe,
           TitlePipe,
           RelativeDatePipe,
-          FastActionButtonComponent
+          FastActionButtonComponent,
+          SenderComponent,
+          ReportVerifyInvalidIconComponent,
```

Comment:
I am curious about why we needed to add these here?  🤔  Seems like the only new code flows we should be hitting involve the SearchTelemetryService, but I guess maybe the existing tests were not covering everything?

Reply:
Because opening a report renders these two components on the page and the test runner was complaining about "`<mm-sender>` is not a known element"